src/components/pages/GamePlay/GameWriteStep.js

```diff
@@ -150,11 +151,12 @@ export default function GameWriteStep(props) {
                 : "I'd rather draw"}
             </button>
 
-            {props.battleReady && (
+            {/* This button is no longer needed as the page routes to next steps. If in the future a battle mode is implemented, this button will be used. */}
```

Comment:
This button is no longer needed as page auto routes to Next Steps with closing of modal.